app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] It is a requirement that the referential integrity for the user model must be set up. Kindly do so
- Check out the given requirements.
![image](https://user-images.githubusercontent.com/81932478/185360577-546bf6b7-fa35-459d-bb0b-518910cddfcd.png)